src/vaults/mainnet.json

```diff
@@ -378,6 +378,12 @@
       "slug": "etherfi-primeliquidberaeth",
       "stakeTokenAddress": "0xb83742330443f7413dbd2abdfc046db0474a944e"
     },
+    {
+      "beraRewardsVault": "0xab1ecd0d1b6730c059a686115741cb131f804368",
+      "depositTokenAddress": "0xaf3ab50bdaea7acfa07fad209bbad504f28c8bcf",
+      "slug": "euler-ehoney",
+      "stakeTokenAddress": "0xaf3ab50bdaea7acfa07fad209bbad504f28c8bcf"
```

Comment:
`0xaf3ab50bdaea7acfa07fad209bbad504f28c8bcf` is not the address of any token. Should this be `0x027DcAfB223f69d41Bd413C50854017718419585`? (ehoney)

Reply:
Something is misconfigured on the tokens then.

```
0xab1ecd0d1b6730c059a686115741cb131f804368 does not match the on-chain rewards vault address for 0x027dcafb223f69d41bd413c50854017718419585. It should be 0x4426220787Fb7C195677BC1753C14E84f9a7692C
```

Reply:
We get this address from the on-chain function `getVault` on `rewardsVaultFactoryAbi`. So the token's `getVault` is returning the deprecated vault.